src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -454,101 +454,85 @@ def _make_cloud_service_type_key(resource_data):
     def _make_region_key(resource_data, provider):
         return f'{resource_data["domain_id"]}.{provider}.{resource_data["region_code"]}'
 
-    def _change_metadata_path(self, metadata):
-        plugin_id = self.transaction.get_meta('plugin_id')
-
-        if plugin_id:
+    @staticmethod
+    def _convert_metadata(metadata, provider):
```

Comment:
The logic to insert custom into the provider is implemented in the previous step.

Reply:
have reflected thank you.